seekers/forms.py

```diff
@@ -0,0 +1,63 @@
+from django import forms
+from .models import Seeker
+
+
+class SeekerCreationForm(forms.ModelForm):
+
+    class Meta:
+        model = Seeker
+        fields = (
+            'city',
+            'start_date',
+            'min_rent',
+            'max_rent',
+            'num_of_roomates',
+            'num_of_rooms',
+            'about',
+        )
+
+    def clean_min_rent(self):
+        min_rent = self.cleaned_data.get('min_rent')
+
+        if not min_rent:
+            raise forms.ValidationError('must have min rent')
```

Comment:
see https://docs.djangoproject.com/en/3.2/ref/forms/fields/
for enforcing required fields without needing to have an if statement for each one